rosdep/python.yaml

```diff
@@ -612,6 +612,33 @@ python-cairo:
     slackpkg:
       packages: [pycairo]
   ubuntu: [python-cairo]
+python-can:
+  alpine:
+    pip:
+      packages: [python-can]
+  arch:
+    pip:
+      packages: [python-can]
+  debian:
+    '*':
```

Comment:
Can you please reverse the rule here ? add a pip key for jessie explicitly and use the widlcard for the other platforms (this is more futureproof, otherwise future debian version will default to the pip rule)

Reply:
Sure, all done.